docs/provisioners/kustomize.md

```diff
@@ -0,0 +1,354 @@
+# kustomize Provisioner
+
+## Summary
+
+The `kustomize` provisioner is one of the [provisioners](https://github.com/operator-framework/rukpak/tree/main/internal/provisioner) of RukPak.
+It instantiates a given `kustomize+v0` bundle onto a cluster and then install the generated resources
```

Comment:
What is a kustomize+v0 bundle?

Reply:
I'll add a short statement with a link to the kustomize page.